test/handlers/AnchorHandler/persist.test.ts

```diff
@@ -32,7 +32,7 @@ describe('AnchorHandler.persist()', () => {
     );
     const sinonMock = sinon.createStubInstance(AuxiliaryChainRepository,
       {
-        update: update as any,
+        save: update as any,
```

Comment:
I would propose to update all tests and not only where compiler complains with regards to interface changes in this PR. Like in this case I would assume stub should be named as `save`. Thoughts?

Reply:
Changes done!